HACKING.md

```diff
@@ -108,8 +112,55 @@ otherwise an official image can be run with
 
     $ make run
 
+Log in to the FAF container
+
+    $ make sh
+
+Set up the database in the `[Storage]` section of `/etc/faf/faf.conf` as follows:
+
+    [Storage]
+    DBUser = faf
+    DBPasswd = scrt
+    DBHost = xxx.xxx.xxx.xxx
+    DBPort = 5432
+    DBName = faf
+
+Use the IP address found above for `DBHost`.
+
 FAF should be available at ```http://localhost:8080/faf/```
 
+#### Web UI with Celery
+
+Download redis docker image
+
+    $ docker pull redis:latest
+
+Start redis container with hostname and opened port 6379:
+
+    $ docker run --name faf-redis --hostname faf-redis -dit -p 6379:6379 redis
+
+Create faf-network and connect faf and faf-redis containers to it:
+
+    $ docker network create faf-network
+    $ docker network connect faf-network faf
+    $ docker network connect faf-network faf-redis
+
+Change hostname in Celery configuration to point to the faf-redis container:
+
+    $ docker exec faf sed -ie 's/localhost/faf-redis/g' /etc/faf/plugins/celery_tasks.conf
+
+Restart faf container
+
+    $ docker restart faf
+
+Start the Celery worker service in the host to enable manual action execution from the UI
```

Comment:
s/in/on/

I don't think this is what we want with containers.

Reply:
```bash
#!/bin/sh

source /etc/faf/celery-beat-env.conf

case $1 in
start)
        /usr/bin/python3 -m celery multi start $CELERYD_NODES \
        -A $CELERY_APP --pidfile=${CELERYD_PID_FILE} \
        --logfile=${CELERYD_LOG_FILE} --loglevel="${CELERYD_LOG_LEVEL}" \
        $CELERYD_OPTS &
        ;;
esac
```

```bash
#!/bin/sh

source /etc/faf/celery-worker-env.conf

case $1 in
start)
        /usr/bin/python3 -m celery multi start $CELERYD_NODES \
        -A $CELERY_APP --pidfile=${CELERYD_PID_FILE} \
        --logfile=${CELERYD_LOG_FILE} --loglevel="${CELERYD_LOG_LEVEL}" \
        $CELERYD_OPTS &
        ;;
stop)
        /usr/bin/python3 -m celery multi stopwait $CELERYD_NODES \
        --pidfile=${CELERYD_PID_FILE} &
        ;;
reload)
        /usr/bin/python3 -m celery multi restart $CELERYD_NODES \
        -A ${CELERY_APP} --pidfile=${CELERYD_PID_FILE} \
        --logfile=${CELERYD_LOG_FILE} --loglevel="${CELERYD_LOG_LEVEL}" \
        $CELERYD_OPTS &
        ;;
esac
```

Reply:
This could replace those `services`.

Reply:
Added the two "service" scripts in the container. The scheduler is still broken though, even though the `beat` service is running. Will look into it.